Zappy_IA_Report/NN/Code/Tutorial7.py

```
@@ -0,0 +1,18 @@
+#python version 3.8
+from keras import *
+
+model = Sequential()
+
+model.add(layers.Dense(units=3, input_shape=[1]))
+model.add(layers.Dense(units=64))
+model.add(layers.Dense(units=1))
+
+x_input = [1,2,3,4,5]
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)